tests/test_events.py

```diff
@@ -188,6 +194,11 @@ def setUp(self):
         self.user = factories.UserFactory()
         self.consolidate_auth = Auth(user=self.user)
         self.project = factories.ProjectFactory()
+
+        # Remove default subscriptions
+        for sub in NotificationSubscription.find():
```

Comment:
Rather than having to do this a bunch of times for different tests, you could change the `setUpClass` [in the base class](https://github.com/CenterForOpenScience/osf.io/blob/develop/tests/base.py#L117), possibly by adding a setting that would disable the creation of these `NotificationSubscriptions` when a `Node` is created. That way you could just re-enable it during `setUp` of notification-type test classes and not have to alter so many different test files.